pkg/project/registry/projectrequest/delegated/delegated.go

```diff
@@ -96,7 +94,7 @@ func (r *REST) Create(ctx kapi.Context, obj runtime.Object) (runtime.Object, err
 		}
 	}
 
-	list, err := r.openshiftClient.TemplateConfigs(r.openshiftNamespace).Create(template)
```

Comment:
> I thought the namespace you submitted to showed up in the objects in the resulting list... guess not

Nope.  I think they really wanted to have both namespaced and non-namespaced templates, but couldn't actually do that.